### ISSUE TITLE
Can my app force the scrollbar to the top of the datagrid?

### ISSUE THREAD

**Issue by runchman** [adazzle/react-data-grid#693]
> **HAVE YOU ALREADY SEARCHED FOR SIMILAR ISSUES? PLEASE HELP US OUT AND DOUBLE-CHECK FIRST!**

> **ALSO, PLEASE DON'T BE THAT PERSON WHO DELETES THIS TEMPLATE. IT'S HERE FOR A REASON.**

> **THANKS!**

**WHICH VERSION OF REACT ARE YOU USING?**
```
Officially Supported:
[ ] v0.14.x
```

```
Community Supported:
[ x] v15.0.x
```

**WHICH BROWSER ARE YOU USING?**

```
Officially Supported:
[ ] IE 9 / IE 10 / IE 11
[ ] Edge
[x ] Chrome
```

```
Should work:
[ ] Firefox
[ ] Safari
```

--------

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[x ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**

**Expected/desired behavior**

**Reproduction of the problem**
If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem.

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

Not an issue per se, but a question: Is there a way to force the scroll bar to the top?

In other words if the user has scrolled down in the datagrid, and then something else happens in my app, I want to trigger the scrollbar to go back to the top of the datagrid.

Possible?
